README.md

```diff
@@ -149,140 +194,52 @@ totalDifficulty      0
-
-  -h, --help
-          Print help (see a summary with '-h')
-```
-
-#### Example Usage
-```bash
-../foundry-zksync/target/debug/zkcast zkdeposit 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 1000000 --rpc-url http://localhost:8545 --l2-url http://localhost:3050 --private-key 7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 --chain 270
-```
-#### Output
-```js
-Bridging assets....
-Transaction Hash: 0x55793df0a636aedd098309e3487c6d9ec0910422d5b9f0bdbdf764bc82dc1b9f
-```
 ---
 
-### ***L2 → L1 withdrawals:***
-
-```bash
-zkcast zk-send --withdraw <TO> --amount <AMOUNT> <TOKEN> --rpc-url <RPC-URL> --private-key <PRIVATE-KEY>
+## Compile with `zkforge zk-build`
 
+> Aliases: `zkforge zkbuild`, `zkforge zk-compile`, `zkforge zkb`.
 
-Arguments:
-  [TO]                  The withdraw recipient.
-
-
-Bridging options:
-  -w, --withdraw        For L2 -> L1 withdrawals.
-
-      --token <TOKEN>   Token to bridge. Leave blank for ETH.
-
-  -a, --amount <AMOUNT> Amount of token to bridge. Required value when bridging
-```
-
-
-#### Example Usage:
-```bash
-../foundry-zksync/target/debug/zkcast zk-send --withdraw 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 --amount 1000000 --rpc-url http://localhost:3050 --private-key 7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 --chain 270
-```
-#### Output
-```js
-Bridging assets....
-Transaction Hash: 0x94ef9e2eed345dcfef6f0b4f953f431b8edc6760e29b598a7cf446dab18d6317
-```
-
----
+Compile smart contracts to zkEvm bytecode and store the compiled output files in a logical directory structure `<PROJECT-ROOT>/zkout/` for easy retrieval by other components of the application.
 
-## Compilation
+```sh
 
-***v0.0*** ***Command***:
-## `zkforge zk-build`
-### aliases: `zkforge zkbuild`, `zkforge zk-compile`, `zkforge zkb`
-
-Compile smart contracts to zkEvm bytecode and store compile output files into a logical directory structure `<PROJECT-ROOT>/zkout/` for easy retrieval for other components of the application.
-
-
-
-
-```bash
 Compiler subcommands for zkSync
 
 Usage: 
 zkforge zk-build [OPTIONS]
 
-
 Options:
       --use-zksolc   Specify zksolc compiler version (default if left blank)
       --is-system    Enable the system contract compilation mode.
       --force-evmla  Sets the EVM legacy assembly pipeline forcibly
       -h, --help         Print help
-
-  
 ```
 
-### Compiling with `--is-system` flag
-It is necessary for some contracts, including those that deploy other contracts (such as factory contracts), to be compiled using the `--is-sytem` flag. These contracts should be placed in the `src/is-system/` folder, if the folder does not exist, manually create it:
+> `--is-system` flag: It is necessary to compile some contracts, including those that deploy other contracts (such as factory contracts), using the `--is-sytem` flag. These contracts should be placed in the `src/is-system/` folder. If the folder does not exist, manually create it.
```

Comment:
typo on the second --is-sytem (missing "s")

Reply:
there were a few of these, updated.